app/templates/views/new-password.html

```diff
@@ -35,7 +35,7 @@ <h1 class="heading-large">
                 "text" : form.password.errors[0] | safe
           } if form.password.errors[0] else None,
         }) }}
-        {{ page_footer("Continue") }}
+        {{ page_footer("Continue", button_name="continue") }}
```

Comment:
I think these are showing up from Jonathans PR, it should make it smaller when he merges later on